src/implementations/twig/components/ecl-component-icon/package.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "@ecl/ec-twig-component-icon",
+  "name": "@ecl/ecl-component-icon",
```

Comment:
Renaming packages, files and folders is not in the scope of this ticket. It's in FRONT-1658
I don't mind having the two, but I do not agree with pattern provided. `@ecl/ecl-` as it's clearly an unnecessary repetition.
We can open the discussion on naming here or leave it for FRONT-1658. Up to you.

Reply:
I personally would like to keep implementation specific path in the pattern: i.e. do not remove `twig`. We'll surely need a separation of base/vanilla and thus removing removing implementation part is unnecessary step.

Reply:
you are right two times, this was actually a mistake, the idea was to have them as ecl-{implementation}-{component}-{componentName} like i did for the button, i will revert those changes keeping the package names as they were.